crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -1119,13 +1138,14 @@ impl SlidingSync {
             // happens here.
             let sync_response = this.inner.client.process_sliding_sync(&response).await?;
 
-            debug!("Sliding sync response has been processed");
+            debug!(?sync_response, "Sliding Sync response has been processed");
```

Comment:
Perhaps consistent verbs here would clarify things further.
```suggestion
            debug!(?sync_response, "Sliding Sync response has been handled");
```

I don't care if we use "handled" or "processed", lets just stick to one. So if you prefer the other one feel free to change things up.

Reply:
Yeah, but “processed” here is different. I'll try to find a better phrasing entirely.